test_run_handler.go

```diff
@@ -67,16 +74,19 @@ func handlePost(w http.ResponseWriter, r *http.Request) {
 	}
 	testRun.CreatedAt = time.Now()
 
-	fmt.Fprintf(w, "got... %v", testRun)
```

Comment:
Note that this output is (was) a bit _debuggy_. The output is now just the JSON blob as stored in the DB, which is a pretty standard behaviour in a REST API.